docs/en/latest/tutorials/monitor-api-health-check.md

```diff
@@ -0,0 +1,180 @@
+Date: Sat, 22 Jul 2023 10:16:38 GMT
+Server: APISIX/3.3.0
+
+hello web2
+
+...
+
+HTTP/1.1 200 OK
+Content-Type: text/plain; charset=utf-8
+Content-Length: 10
+Connection: keep-alive
+Date: Sat, 22 Jul 2023 10:16:39 GMT
+Server: APISIX/3.3.0
+
+hello web1
+```
+
+## Collecting health check data with the Prometheus plugin
+
+Once the health checks and route are configured in APISIX, you can employ Prometheus to monitor health checks. APISIX **automatically exposes health check metrics data** for your APIs if the health check parameter is enabled for upstream nodes. You will see metrics in the response after fetching them from APISIX:
+
+```bash
+curl -i http://127.0.0.1:9091/apisix/prometheus/metrics
+```
+
+Example Output:
+
+```bash
+# HELP apisix_http_requests_total The total number of client requests since APISIX started
+# TYPE apisix_http_requests_total gauge
+apisix_http_requests_total 119740
+# HELP apisix_http_status HTTP status codes per service in APISIX
+# TYPE apisix_http_status counter
+apisix_http_status{code="200",route="1",matched_uri="/",matched_host="",service="",consumer="",node="172.27.0.5"} 29
+apisix_http_status{code="200",route="1",matched_uri="/",matched_host="",service="",consumer="",node="172.27.0.7"} 12
+# HELP apisix_upstream_status Upstream status from health check
+# TYPE apisix_upstream_status gauge
+apisix_upstream_status{name="/apisix/upstreams/1",ip="172.27.0.5",port="443"} 0
+apisix_upstream_status{name="/apisix/upstreams/1",ip="172.27.0.5",port="80"} 1
+apisix_upstream_status{name="/apisix/upstreams/1",ip="172.27.0.7",port="443"} 0
+apisix_upstream_status{name="/apisix/upstreams/1",ip="172.27.0.7",port="80"} 1
+```
+
+Health check data is represented with metrics label `apisix_upstream_status`. It has attributes like upstream `name`, `ip` and `port`. A value of 1 represents healthy and 0 means the upstream node is unhealthy.
+
+## Visualize the data in Prometheus dashboard
+
+Navigate to http://localhost:9090/ where Prometheus instance is running in Docker and type **Expression** `apisix_upstream_status` in the search bar. You can also see the output of the health check statuses of upstream nodes on the **Prometheus dashboard** in the table or graph view:
+
+![Visualize the data in Prometheus dashboard](https://static.apiseven.com/uploads/2023/07/20/OGBtqbDq_output.png)
+
+## Cleanup
+
+Once you are done experimenting with Prometheus and APISIX Gateway health check metrics, you can use the following commands to stop and remove the services created in this guide:
+
+```bash
+docker compose down
+```
+
+## Next Steps
+
+You have now learned how to setup and monitor API health checks with Prometheus and APISIX.  APISIX Prometheus plugin is configured to connect [Grafana](https://grafana.com/) automatically to visualize metrics. Keep exploring the data and customize [Grafana dashboard](https://grafana.com/grafana/dashboards/11719-apache-apisix/) by adding a panel that shows the number of active health checks.
```

Comment:
You can also reference your previous blog post: https://dev.to/apisix/implementing-resilient-applications-with-api-gateway-health-check-338c

Reply:
Sure, I put some references.